orte/mca/oob/tcp/oob_tcp.c

```diff
@@ -301,8 +301,7 @@ static void process_set_peer(int fd, short args, void *cbdata)
                         ORTE_NAME_PRINT(&pop->peer),
                         (NULL == pop->net) ? "NULL" : pop->net,
                         (NULL == pop->port) ? "NULL" : pop->port);
-    maddr = OBJ_NEW(mca_oob_tcp_addr_t);
-    memcpy(&maddr->addr, &inaddr, sizeof(inaddr));
+    /* memcpy(&maddr->addr, &inaddr, sizeof(inaddr)); */
```

Comment:
Why leave this line commented out here? If it needs to be deleted, then let's delete it.